Formula/botan.rb

```diff
@@ -29,6 +29,8 @@ def install
       --cpu=#{MacOS.preferred_arch}
       --cc=#{ENV.compiler}
       --os=darwin
+      --with-external-includedir=#{Formula["openssl@1.1"].include}
```

Comment:
These are unnecessary and will break the build since then it won't be able to find any other headers. Ditto the `--with-external-libdir` below.